src/main/java/com/amihaiemil/docker/ListedVolumes.java

```diff
@@ -74,22 +78,36 @@
         final FilteredUriBuilder uri = new FilteredUriBuilder(
             new UncheckedUriBuilder(super.baseUri().toString()), this.filters
         );
-        return new ResourcesIterator<>(
-            super.client(),
-            new HttpGet(
-                uri.build()
-            ),
-            volume -> new RtVolume(
-                volume,
-                super.client(),
-                URI.create(
-                    String.format("%s/%s",
-                        super.baseUri().toString(),
-                        volume.getString("Name")
+        final HttpGet get = new HttpGet(uri.build());
+        try {
+            final JsonObject obj = super.client().execute(
```

Comment:
@bkuzmic You don't need to create this `obj` variable here, it is just used once